spec/koans.js

```diff
@@ -301,29 +319,29 @@ describe('destructuring objects. ', () => {
 describe('destructuring can also have default values. ', () => {
 
   it('for an empty array', () => {
-    const [a] = [];
-    //expect(a).toEqual(1)
+    const [a] = [1];
+    expect(a).toEqual(1)
   });
 
```

Comment:
Ni en esta ni en las siguientes habéis aprovechado el valor de los valores por defecto. 

![image](https://user-images.githubusercontent.com/26923217/73741284-32950380-474a-11ea-8f72-de75a045b7d7.png)